DeviceId.js

```diff
@@ -0,0 +1,75 @@
+import * as L from "./Logger.js";
+import * as Validate from "./Validators.js";
+import * as Utils from "./Utils.js";
+
+class DeviceId {
+    #state;
+
+    constructor(state) {
+        this.#state = state;
+    }
+
+    /**
+     *
+     * Get currently used device Id.
+     * Should be called after Countly init
+     *
+     * @return {string} device id or error message
+     */
+    getCurrentDeviceId = async function () {
```

Comment:
overall it is a nice approach however I recommend to comply with Android and iOS lets use same function naming with Android. https://github.com/Countly/countly-sdk-android/blob/15271104c1c7cd772608e9f16289914e4ab880e4/sdk/src/main/java/ly/count/android/sdk/ModuleDeviceId.java#L298